### ISSUE TITLE
Kobowave

### ISSUE THREAD

**Issue by EpicGermanLeader** [jogerj/place-atlas#13]
Paste the atlas code output in the codeblock below:
```    {
    	"id": 0,
    	"name": "kobowave",
    	"description": "a tribute to Vtuber Kobo Kanaeru from Hololive ID.",
    	"website": "",
    	"subreddit": "r/hololive",
    	"center": [
    		1915.5,
    		714.5
    	],
    	"path": [
    		[
    			1891.5,
    			696.5
    		],
    		[
    			1940.5,
    			696.5
    		],
    		[
    			1936.5,
    			731.5
    		],
    		[
    			1940.5,
    			727.5
    		],
    		[
    			1940.5,
    			732.5
    		],
    		[
    			1891.5,
    			732.5
    		],
    		[
    			1891.5,
    			714.5
    		],
    		[
    			1891.5,
    			696.5
    		],
    		[
    			1940.5,
    			696.5
    		],
    		[
    			1940.5,
    			732.5
    		],
    		[
    			1891.5,
    			732.5
    		]
    	]
    }

```

**Comment by rob-not-bob** [comment]
Suggested website. Her channel: https://www.youtube.com/channel/UCjLEmnpCNeisMxy134KPwWw

**Comment by jogerj** [comment]
Resolved in #14
